Scaffold barebone ExpressJS application

titu opened the pull request.

titu committed:
Scaffold barebone ExpressJS application

arealmaas commented on app.js:
```diff
@@ -0,0 +1,46 @@
+var express = require('express');
```
Since we're using es6, we can use `const` for modules
> var express = require('express');

const express = require('express');

arealmaas commented on bin/www:
```diff
@@ -0,0 +1,90 @@
+ * Module dependencies.
+ */
+
+var app = require('../app');
```
use const wherever possible :) Inside the functions as well

arealmaas commented on bin/www:
```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env node
```
Just out of curiosity, why don't you prefix this file with .js, and instead specify it as a node executable or whatever you make it in a she-bang?

titu replied:
Express generator doing this :p . I think renaming by adding the .js extension is a good idea, will do that.

arealmaas replied:
cool!

arealmaas commented on public/stylesheets/style.css:
```diff
@@ -0,0 +1,8 @@
+body {
```
Should have specified a bit more that this is a API only and no client will be rendered. Sorry about that :) Can just remove the html-rendering logic / libraries

titu committed:
Changes made based on the code review:

1. Replaced 'var' with 'const' / 'let'
2. Renamed the /bin/www to /bin/www.js
3. View logic/files removed
